.rubocop.yml

```diff
@@ -2,7 +2,7 @@ inherit_gem:
   jekyll: .rubocop.yml
 
 AllCops:
-  TargetRubyVersion: 2.2
+  TargetRubyVersion: 2.4
```

Comment:
As they were no differences for this plugin, I enforced 2.4.

Reply:
But won't this allow Rubocop to make suggestions that would be incompatible with Ruby 2.3? I don't think we want this.

Reply:
Fair. Reverted.